docusaurus.config.ts

```diff
@@ -154,11 +154,15 @@ const config: Config = {
         path: 'theoplayer',
         routeBasePath: '/theoplayer',
         sidebarPath: './sidebarsTheoplayer.ts',
-        lastVersion: 'current',
+        lastVersion: 'v9',
+        includeCurrentVersion: !isProductionDeployment,
```

Comment:
☝️ This is the important bit. The current version (v10) will not show up in production builds, but they'll still show in local builds and PR previews.